src/java/generator/MapGenerator.java

```diff
@@ -421,55 +447,55 @@ private static void parseMapName() {
 	}
 
 	private static void randomizeOptions(){
-		SPAWN_COUNT = 6;
-		LAND_DENSITY = StrictMath.min((RANDOM.nextInt(127)+13.0f)/127,1);
-		PLATEAU_DENSITY = (float) RANDOM.nextInt(127)/127*0.2f;
-		MOUNTAIN_DENSITY = (float) RANDOM.nextInt(127)/127*0.075f;
-		RAMP_DENSITY = (float) RANDOM.nextInt(127)/127*0.2f;
-		RECLAIM_DENSITY = (float) RANDOM.nextInt(127)/127;
-		MEX_COUNT = 8 + 4/(SPAWN_COUNT/2) + RANDOM.nextInt(40/SPAWN_COUNT);
+		spawnCount = 6;
+		landDensity = StrictMath.min((random.nextInt(127)+13.0f)/127,1);
+		plateauDensity = (float) random.nextInt(127)/127*0.2f;
+		mountainDensity = (float) random.nextInt(127)/127*0.075f;
+		rampDensity = (float) random.nextInt(127)/127*0.2f;
+		reclaimDensity = (float) random.nextInt(127)/127;
+		mexCount = 8 + 4/(spawnCount /2) + random.nextInt(40/ spawnCount);
 	}
 
 	private static void parseOptions(byte[] optionBytes){
 		if (optionBytes.length>0){
 			if (optionBytes[0]<=16){
-				SPAWN_COUNT = optionBytes[0];
+				spawnCount = optionBytes[0];
 			}
 		}
 		if (optionBytes.length>1){
-			LAND_DENSITY = (float) optionBytes[1]/127+13.0f/127;
+			landDensity = (float) optionBytes[1]/127+13.0f/127;
 		}
 		if (optionBytes.length>2){
-			PLATEAU_DENSITY = (float) optionBytes[2]/127*0.2f;
+			plateauDensity = (float) optionBytes[2]/127*0.2f;
 		}
 		if (optionBytes.length>3){
-			MOUNTAIN_DENSITY = (float) optionBytes[3]/127*0.075f;
+			mountainDensity = (float) optionBytes[3]/127*0.075f;
 		}
 		if (optionBytes.length>4){
-			RAMP_DENSITY = (float) optionBytes[4]/127*0.2f;
+			rampDensity = (float) optionBytes[4]/127*0.2f;
 		}
 		if (optionBytes.length>5){
-			RECLAIM_DENSITY = (float) optionBytes[5]/127;
+			reclaimDensity = (float) optionBytes[5]/127;
 		}
 		if (optionBytes.length>6){
-			MEX_COUNT = optionBytes[6];
+			mexCount = optionBytes[6];
```

Comment:
spacing for all binary operators (divison, multiplication and comparators)

Reply:
Done